httpclient5/src/main/java/org/apache/hc/client5/http/impl/auth/SPNegoScheme.java

```diff
@@ -41,14 +41,7 @@
  * </p>
  *
  * @since 4.2
- *
- * @deprecated Do not use. The GGS based experimental authentication schemes are no longer
- * supported. Consider using Basic or Bearer authentication with TLS instead.
- *
- * @see BasicScheme
- * @see BearerScheme
  */
-@Deprecated
 @Experimental
 public class SPNegoScheme extends GGSSchemeBase {
```

Comment:
Should also go in favor of `SpnegoScheme`

Reply:
OK